### ISSUE TITLE
Script not detecting new Inkscape window (Ubuntu 22)

### ISSUE THREAD

**Issue by vikomen** [gillescastel/inkscape-shortcut-manager#29]
I have followed the instructions exactly and I have been able to replicate all of the results to a virtual machine ruining Cutefish OS (Linux) , and to windows wsl 2. I decided that I wanted to dualboot my laptop and I installed Ubuntu 22. I did the exact same installation progress and I got the inscape-figure thing to work, however, I can't get this script to work as well. Whenever I run it, I get

`vikomen@VikoLaptop:~/inkscape-shortcut-manager-master$ python3 main.py
{'style': PosixPath('/home/vikomen/.config/inkscape-shortcut-manager/styles'), 'object': PosixPath('/home/vikomen/.config/inkscape-shortcut-manager/objects')}

`
as on the other devices. However, when I open a new terminal window and run inkscape, the script does not seem to detect it.  Has anyone experienced this? I can add a video if that would help.

**Comment by pabloavi** [comment]
Hi.

Whenever I tried to execute main.py, I got the output 

`{'style': PosixPath('/home/pablo/.config/inkscape-shortcut-manager/styles'), 'object': PosixPath('/home/pablo/.config/inkscape-shortcut-manager/objects')}`,

which I guess was OK. However, opening a new inkscape window didn't make anything, nor where the shortcuts working. I realized Wayland window manager may be incompatible with the script, so I installed Xorg, started the session with it, and everything successfully worked: the initial output was the mentioned, but now it detected the inkscape windows and shortcuts were working.

That being said, as you don't give more information I can't suggest any other thing than checking whether you're using Waylad and, if so, installing Xorg and starting a session with it.

Hope it helps.
